src/Rings/orderings.jl

```diff
@@ -453,32 +456,34 @@ Given a vector `V` of variables, return the reverse lexicographical ordering on
 julia> R, (w, x, y, z) = polynomial_ring(QQ, ["w", "x", "y", "z"])
 (Multivariate polynomial ring in 4 variables over QQ, QQMPolyRingElem[w, x, y, z])
 
-julia> o1 = revlex(R)
-revlex([w, x, y, z])
+julia> o1 = invlex(R)
+invlex([w, x, y, z])
 
 julia> canonical_matrix(o1)
 [0   0   0   1]
 [0   0   1   0]
 [0   1   0   0]
 [1   0   0   0]
 
-julia> o2 = revlex([w, x])
-revlex([w, x])
+julia> o2 = invlex([w, x])
+invlex([w, x])
 
-julia> o3 = revlex(gens(R)[3:4])
-revlex([y, z])
+julia> o3 = invlex(gens(R)[3:4])
+invlex([y, z])
 ```
 """
-function revlex(R::MPolyRing)
-  return MonomialOrdering(R, SymbOrdering(:revlex, 1:nvars(R)))
+function invlex(R::MPolyRing)
+  return MonomialOrdering(R, SymbOrdering(:invlex, 1:nvars(R)))
 end
 
-function revlex(v::AbstractVector{<:MPolyRingElem})
+function invlex(v::AbstractVector{<:MPolyRingElem})
   i = _unique_var_indices(v)
-  return MonomialOrdering(parent(first(v)), SymbOrdering(:revlex, i))
+  return MonomialOrdering(parent(first(v)), SymbOrdering(:invlex, i))
 end
 
-function _matrix(nvars::Int, o::SymbOrdering{:revlex})
+const revlex = invlex
```

Comment:
Why is this in here? Wasn't the plan to get rid of this meaning of `revlex` (either removing `revlex`, or giving it a new meaning -- though I'd still vote for first removing it, at least for a time, so that code using it can be caught and adapted)
```suggestion
```